scenarios/grpc_search.js

```diff
@@ -0,0 +1,32 @@
+import { check } from 'k6';
+import native from 'k6/x/neofs/native';
+
+const grpc_client = native.connect(__ENV.GRPC_ENDPOINT, '', 5, 15);
+const container = __ENV.cid
+
+export const options = {
+    scenarios: {
+        system_write: {
+            executor: 'shared-iterations',
+            vus: __ENV.vu,
+            iterations: __ENV.i,
+            exec: 'search',
+            maxDuration: (24*365*100).toString()+"h", // default is 10m and this load is designed to be controlled by iterations only
```

Comment:
doesnt zero mean inf?

Reply:
idk, and it is not clear from docs: https://k6.io/docs/using-k6/k6-options/reference/#iterations
we can try what it means in practice. not sure if we can rely on it

Reply:
docs dont state this, so we cannot